apps/web/components/eventtype/EventWebhooksTab.tsx

```diff
@@ -139,7 +137,7 @@ export const EventTeamWebhooksTab = ({
                   <EmptyScreen
                     Icon={TbWebhook}
                     headline={t("create_your_first_webhook")}
-                    description={t("create_your_first_team_webhook_description", { appName: APP_NAME })}
```

Comment:
We should cleanup all common.json as I think it's not being used anywhere.
<img width="589" alt="Screenshot 2023-05-22 at 7 58 10 PM" src="https://github.com/calcom/cal.com/assets/1780212/2adab3e2-b460-488c-8995-c3d049e84aae">